beacon_node/execution_layer/src/test_utils/mod.rs

```diff
@@ -600,7 +600,7 @@ async fn handle_rejection(err: Rejection) -> Result<impl warp::Reply, Infallible
     let message;
 
     if let Some(e) = err.find::<AuthError>() {
```

Comment:
nit:
```suggestion
    if let Some(AuthError(e)) = err.find::<AuthError>() {
        message = format!("Authorization error: {}", e);
```

I couldn't include the delete line in the suggestion so had to make it to this one sorry